ledger/complete/mtrie/forest.go

```diff
@@ -50,7 +50,7 @@ func NewForest(forestCapacity int, metrics module.LedgerMetrics, onTreeEvicted f
 			if !ok {
 				panic(fmt.Sprintf("cache contains item of type %T", value))
 			}
-			//TODO Log error
+			// TODO Log error
```

Comment:
lets remove it

Reply:
Removed in 0ee96753c82d52d49152860d19ff44484a98ddec.  

I'll open separate issue/pr to improve logging.  It will add logging of this error and some other warnings/errors/etc.